funnel/forms/space.py

```diff
@@ -92,6 +92,16 @@ class ParticipantForm(forms.Form):
         validators=[forms.validators.DataRequired(u"Select at least one event")])
 
 
+class ParticipantImportForm(forms.Form):
+    participant_list = forms.FileField(__("Participant List"),
+        description=u"CSV with Header. Order: name, email, phone, twitter, company.",
```

Comment:
Why do you need headers if you need a specific order? Pick one. See Hasmail for how it's handled there if you need ideas.